src/harness/harnessIO.ts

```diff
@@ -1063,7 +1063,12 @@ export namespace Compiler {
         tsSources: readonly TestFile[],
         prettyErrors: boolean | undefined,
         reason: string | undefined,
+        expectNull?: boolean,
     ) {
+        if (expectNull) {
+          // eslint-disable-next-line no-null/no-null
+          return Baseline.runBaseline(type + "/" + baselinePath.replace(/\.tsx?/, `.d.ts.diff`), /*actual*/ null);
+        }

```

Comment:
Why was this necessary ?

Reply:
So we were polluting the repo by keeping files that aren't being tested against references (when tsc and dte results were different before thus checked-in to the repo but later when they became identical) and creating confusion because we weren't testing it, it took me some time to understand why the tests weren't complaining about the files that shouldn't be present.